webview/src/experiments/components/table/Table.tsx

```diff
@@ -205,6 +219,7 @@ export const Table: React.FC<TableProps & WithChanges> = ({
             hasRunningExperiment={hasRunningExperiment}
             projectHasCheckpoints={hasCheckpoints}
             batchRowSelection={batchRowSelection}
+            root={tableRef.current}
```

Comment:
Since the experiment column's cells needed access to `tableRef.current`, I ended up needing to carry the `root` prop across multiple table components. Though maybe there is an easier way that doesn't involve having to touch so many components 🤔